rickchurch/church.py

```diff
@@ -61,6 +64,8 @@ def custom_openapi() -> Dict[str, Any]:
 @app.on_event("startup")
 async def startup() -> None:
     """Create asyncpg connection pool on startup and setup logging."""
+    app.state.client = AsyncClient()
```

Comment:
Maybe call this something like `httpx_client`, since `client` is already used by `pydispix.Client` and it's a bit confusing